### DISCUSSION TITLE
Support for Claude 5 Sonnet with Amazon Bedrock

### DISCUSSION THREAD

**Discussion by jakedreamsec** [ScrapeGraphAI/Scrapegraph-ai#518] [Ideas]
Will ScrapegraphAI support Claude 3.5 Sonnet with Amazon bedrock in the near future? That would be helpful

**Comment by VinciGit00** [comment]
just added

**Reply by elibaskin** [reply]
Oh my, I was just looking for support for Claude! What a coincidence! 

Does it work directly with Anthropic?
